Source/Global/global.cpp

```diff
@@ -17,12 +17,18 @@ static std::shared_ptr<http_singleton> g_httpSingleton_atomicReadsOnly;
 
 NAMESPACE_XBOX_HTTP_CLIENT_BEGIN
 
-http_singleton::http_singleton(IHCPlatformContext* initialContext) :
-    m_platformContext{ initialContext }
+http_singleton::http_singleton(PerformInfo const& performInfo, PerformEnv&& performEnv) :
+    m_callRoutedHandlersContext{ 0 },
+    m_perform{ performInfo },
+    m_performEnv{ std::move(performEnv) },
+    m_lastId{ 0 },
+    m_retryAllowed{ false },
```

Comment:
>m_retryAllowed{ false } [](start = 4, length = 23)

You're switching the default value for this. Is that intentional? Do you have to turn this back on somewhere that I'm missing?

Reply:
I think I started from older code that defaulted to retries off